.circleci/config.yml

```diff
@@ -38,7 +38,7 @@ jobs:
                   path: ~/repo/packages/abi-gen/test-cli/output
```

Comment:
git diff does a crap job w/this file so I'l summarize what went on:
- CI was running out of memory in `test-contracts-rest` so I moved staking to its own job
- I removed the `3.0` suffixes